analyses/cell-type-ewings/exploratory_analysis/annotation_notebooks/SCPCS000752/SCPCL001114_tumor-gene-expression-exploration.Rmd

```diff
@@ -0,0 +1,522 @@
+  geom_density() +
+  facet_grid(rows = vars(geneset))
+```
+
+Again we see that the gene set scores are bimodal for `SCPCL000822`, at least for `RIGGI` and `ZHANG`. 
+However, we don't see that for `SCPCL001114`, but we do see that the majority of the distribution seems to line up with the upper distribution in `SCPCL000822`.
+Or at least this is the case for `RIGGI` and `ZHANG`.
+This makes me think that the majority of the cells are in fact tumor cells. 
+
+## Comparison to other classification methods 
+
+This section will compare using the marker gene cutoff determined from `SCPCL000822` to other classification methods. 
+First we will compare the calls between each individual method and the marker gene method. 
+Then, we will look at all methods together and plot the expression of marker genes and gene set scores for normal and tumor cells. 
+
+### `AUCell`
+
+[`AUCell`](https://www.bioconductor.org/packages/release/bioc/vignettes/AUCell/inst/doc/AUCell.html) was run previously using the `aucell-annotation.sh` workflow. 
+In that workflow, we ran `AUCell` on both a reference library, `SCPCL000822` and the query library, `SCPCL001114`. 
+The AUC value determined by running `AUCell` with `SCPCL000822` was used as a cutoff for identifying tumor cells in `SCPCL001114`. 
+
+Here we will compare the classifications obtained by using `AUCell`. 
+The classification using marker gene expression from `SCPCL000822` is the rows and the classification using `AUCell` is the columns. 
+
+```{r}
+classification_df <- classification_df |> 
+  dplyr::mutate(
+    ref_marker_gene_classification = forcats::fct_relevel(ref_marker_gene_classification, "Tumor"),
+    auc_classification = forcats::fct_relevel(auc_classification, "Tumor")
+  )
+
+# compare using a confusion matrix
+caret::confusionMatrix(
+    table(
+      classification_df$ref_marker_gene_classification, 
+      classification_df$auc_classification)
+  ) 
+```
+
+It looks like using `AUCell` results in more cells being called normal than with the marker gene cutoff determined by `SCPCL000822`. 
+
+### Expression of at least one marker gene
+
+In the `aucell-annotation.sh` workflow, we looked at marker gene expression and classified any cell that expresses at least 1 marker gene as a tumor cell. 
+Let's compare the annotations obtained here using a cutoff determined by `SCPCL000822` to setting the cutoff of 1 marker gene. 
+
+First we will look at a confusion matrix between both methods. 
+Here the classification using expression of at least 1 marker gene is the rows and the updated classification based on `SCPCL000822` is the columns. 
+
+```{r}
+classification_df <- classification_df |> 
+  dplyr::mutate(
+    marker_gene_classification = forcats::fct_relevel(marker_gene_classification, "Tumor")
+  )
+
+# compare using a confusion matrix
+caret::confusionMatrix(
+    table(
+      classification_df$marker_gene_classification, 
+      classification_df$ref_marker_gene_classification)
```

Comment:
Can you swap the order of these for consistency with the previous plot? I was very confused trying to compare these two for a while.
```suggestion
      classification_df$ref_marker_gene_classification,
      classification_df$marker_gene_classification)
```